### PR TITLE
Simplify delegate invocation

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

**Comment by Pilchie** [comment]
:+1: for the IDE changes.  Tagging @dotnet/roslyn-compiler for the compiler changes.

File: src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs [hunk 1]
```diff
@@ -98,11 +98,7 @@ void ICommandHandlerService.Execute<T>(IContentType contentType, T args, Action
             }
             else
             {
-                // If there aren't any command handlers, just invoke the last handler (if there is one).
-                if (lastHandler != null)
-                {
-                    lastHandler();
-                }
+                lastHandler?.Invoke();
```
Comment (jaredpar) [inline src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs]: Is there a reason we deleted the comment here?

**Comment by jaredpar** [comment]
:+1:

File: src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs (hunk 1 above)
Comment (DustinCampbell) [inline src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs]: Looks like Cyrus's code fix doesn't handle trivia. Hey @cyrusnajmabadi, try using Matt's removenode API that attempts to leave trivia behind.

Comment (jaredpar) [inline src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs]: Ah I didn't realize this was a result of running his fix against our code base.  Like this change even more now.

Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs]: Good call. I'll update my fixer accordingly.

**Pull request merged**